client/state/components/drop-zone/selectors.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * External dependencies
+ */
+import get from 'lodash/get';
+
+export function isDropZoneVisible( state ) {
+	return get( state, 'dropZone.isVisible', false );
+}
```

Comment:
Moved the selector to its new home :) Thanks for pointing me to that discussion!